lib/rules/no-unescaped-entities.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @fileoverview HTML special characters should be escaped.
+ * @author Patrick Hayes
+ */
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+// NOTE: '<' and '{' are also problematic characters, but they do not need
+// to be included here because it is a syntax error when these characters are
+// included accidentally.
+var DEFAULTS = ['>', '"', '\'', '}'];
+
+module.exports = function(context) {
```

Comment:
This should use the [new eslint rule format](http://eslint.org/blog/2016/07/eslint-new-rule-format)